controllers/noobaa-rules.yaml

```diff
@@ -0,0 +1,191 @@
+ groups:
+   - name: noobaa-telemeter.rules
+     rules:
+       - expr: |
+           NooBaa_num_buckets_claims{namespace="redhat-data-federation"}
+         record: noobaa_num_buckets_claims
+       - expr: |
+           NooBaa_num_unhealthy_bucket_claims{namespace="redhat-data-federation"}
+         record: noobaa_num_unhealthy_bucket_claims
+       - expr: |
+           NooBaa_num_unhealthy_namespace_buckets{namespace="redhat-data-federation"}
+         record: job:noobaa_total_unhealthy_namespace_buckets
+       - expr: |
+           NooBaa_num_namespace_buckets{namespace="redhat-data-federation"}
+         record: job:noobaa_namespace_bucket_count
+       - expr: |
+           NooBaa_accounts_num{namespace="redhat-data-federation"}
+         record: noobaa_accounts_num
+       - expr: |
+           NooBaa_total_usage{namespace="redhat-data-federation"}
+         record: noobaa_total_usage
+       - expr: |
+            NooBaa_num_unhealthy_namespace_resources{namespace="redhat-data-federation"}
+         record: noobaa_num_unhealthy_namespace_resources
+       - expr: |
+           NooBaa_num_namespace_resources{namespace="redhat-data-federation"}
+         record: nooBaa_num_namespace_resources
+#       - expr: |
+#           NooBaa_namespace_resource_status{namespace="redhat-data-federation"}
+#         record: noobaa_namespace_resource_status
+#       - expr: |
+#           NooBaa_namespace_bucket_status{namespace="redhat-data-federation"}
+#         record: noobaa_namespace_bucket_status
```

Comment:
Made changes based on @jmolmo's review and a conversation with @cloudbehl on Friday. I've commented this out right now as I need to test these two metrics on a cluster before I incorporate them.

Reply:
@cloudbehl For these two metrics, I'm not getting anything in Prometheus.

<details>
<summary>Screenshots</summary>

![Screenshot from 2022-07-13 13-29-28](https://user-images.githubusercontent.com/33557095/178683616-38147828-9c4f-47da-ad52-791f1a647e6b.png)

![image](https://user-images.githubusercontent.com/33557095/178682278-25a139f8-4003-491c-b44e-fec18cf585de.png)

</details>

Reply:
did you create the namespace bucket  and bucket class to see the availability of the metrics?

Reply:
Yep, I created these and now they are available (as seen in the PDF I sent you). I've added them.